lib/Models/Terria.ts

```diff
@@ -971,26 +971,22 @@ export default class Terria {
       reference.setTrait(CommonStrata.definition, "url", magdaRoot);
       reference.setTrait(CommonStrata.definition, "recordId", id);
       reference.setTrait(CommonStrata.definition, "magdaRecord", config);
-      await reference.loadReference().then(() => {
```

Comment:
@KeyboardSounds to fix #4987, I think we need to remove the check for `groups` and `groups.members` at https://github.com/TerriaJS/terriajs/blob/891aafb9bfb1ceb153b14931073ab7a599a98bde/lib/Models/Terria.ts#L957

Reply:
whoops! it would help if I fixed the actual bug 😂